korean/coco/journey-to-find-coco/find-coco.md

```diff
@@ -0,0 +1,3 @@
+아 근데 같이 먹을 마쉬멜로우를 가져온다는걸 깜빡했네.방으로 
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.